javascript/statements.json

```
@@ -1864,7 +1864,8 @@
                 }
               ],
               "firefox": {
-                "version_added": false
+                "version_added": false,
+                "notes": "See <a href='https://bugzilla.mozilla.org/show_bug.cgi?id=1247687'>bug 1247687</a>"
```

This is failing CI because of the link format. Try this instead:

```suggestion
                "notes": "See <a href='https://bugzil.la/1247687'>bug 1247687</a>."
```